tools/scylla-sstable.cc

```diff
@@ -2937,8 +2938,13 @@ Validate the specified sstables:
 
         std::vector<sstables::shared_sstable> sstables;
         if (app_config.count("sstables")) {
+            const auto sstable_names = app_config["sstables"].as<std::vector<sstring>>();
+            if (std::set(sstable_names.begin(), sstable_names.end()).size() != sstable_names.size()) {
+                fmt::print(std::cerr, "Duplicated sstables specified\n");
```

Comment:
error processing arguments: duplicate sstable arguments found

Reply:
incorporated the proposed error message. thought "duplicate" was a verb but now i see it is also an adjective. thank you! TIL.